ui-v2/src/components/automations/automation-details/automation-details.tsx

```diff
@@ -1,27 +1,58 @@
-import { Automation, buildGetAutomationQuery } from "@/api/automations";
+import { type Automation } from "@/api/automations";
 import { ActionDetails } from "@/components/automations/action-details";
 import { AutomationEnableToggle } from "@/components/automations/automation-enable-toggle";
 import { AutomationsActionsMenu } from "@/components/automations/automations-actions-menu";
 import { useDeleteAutomationConfirmationDialog } from "@/components/automations/use-delete-automation-confirmation-dialog";
+import {
+	Breadcrumb,
+	BreadcrumbItem,
+	BreadcrumbLink,
+	BreadcrumbList,
+	BreadcrumbPage,
+	BreadcrumbSeparator,
+} from "@/components/ui/breadcrumb";
+import { Card } from "@/components/ui/card";
 import { DeleteConfirmationDialog } from "@/components/ui/delete-confirmation-dialog";
 import { Typography } from "@/components/ui/typography";
-import { useSuspenseQuery } from "@tanstack/react-query";
-import { NavHeader } from "./nav-header";
 
-type AutomationPageProps = {
-	id: string;
+type DisplayType = "page" | "item";
+
+type AutomationDetailsProps = {
+	data: Automation;
+	displayType: DisplayType;
+};
+
+export const AutomationDetails = ({
+	data,
+	displayType,
+}: AutomationDetailsProps) => {
+	if (displayType === "item") {
+		return (
+			<Card className="p-4 pt-5">
+				<AutomationDetailsContent data={data} displayType={displayType} />
+			</Card>
+		);
+	}
+	return <AutomationDetailsContent data={data} displayType={displayType} />;
 };
```

Comment:
I think this card wrapping should be handled in the `AutomationsPage` component. I don't think these components should have knowledge of different display modes since it couples the two pages together. That's knowledge that the parent component should have.